scripts/test/Muon/PeriodicTableModel_test.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, print_function
+
+import unittest
+
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table_model import PeriodicTableModel
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+class PeriodicTableModelTest(unittest.TestCase):
+    def setUp(self):
+        self.model = PeriodicTableModel()
```

Comment:
I think we should add a small example file and check that the data is read correctly